renpy/display/transform.py

```diff
@@ -770,12 +770,11 @@ def event(self, ev, x, y, st):
         return None
 
     def __call__(self, child=None, take_state=True, _args=None):
-        child = renpy.easy.displayable_or_none(child)
 
         if child is None:
             child = self.child
 
-        if (child is not None) and (child._duplicatable):
+        if (child is not None) and getattr(child, '_duplicatable', False):
```

Comment:
You can even get rid of the `child is not None` part now. It's great ! Hadn't realized how simple it is now.

Reply:
Good call, overwhelming majority of calls are `not None` so optimising for that case makes sense. 👍